### ISSUE TITLE
Remove v1alpha1 support from cli

### ISSUE THREAD

**Issue by pradeepitm12** [tektoncd/cli#1335]
Currently, both v1alpha1 and v1beta1 are supported. But going forward we will need to remove v1alpha1 support from cli.
We may need to document in one or two releases about the removal of v1alpha1.
Also IMHO it would be nice to have cli support the current and current-1 version.

Let's keep this issue as our discussion point and get feedback from users and contributors.

cc: @tektoncd/cli-collaborators

**Comment by chmouel** [comment]
👍🏻

**Comment by vinamra28** [comment]
PR to track the removal of v1alpha1 APIs https://github.com/tektoncd/pipeline/pull/5005

**Comment by vdemeester** [comment]
This has to be done before bumping 0.38.0, which means before 0.26, hence the priority.

**Comment by vinamra28** [comment]
- [ ] Task
- [ ] ClusterTask
- [ ] Pipeline
- [ ] TaskRun
- [ ] PipelineRun
